sim/dex-data.ts

```diff
@@ -905,70 +911,72 @@ export class Move extends BasicEffect implements Readonly<BasicEffect & MoveData
 		this.stab = data.stab || undefined;
 		this.volatileStatus = typeof data.volatileStatus === 'string' ? (data.volatileStatus as ID) : undefined;
 
-		if (this.category !== 'Status' && !this.gmaxPower) {
+		if (this.category !== 'Status' && !this.maxMove?.basePower) {
+			this.maxMove = this.maxMove || {} as MoveData['maxMove'];
 			if (this.isMax || this.isZ) {
-				this.gmaxPower = 1;
+				this.maxMove!.basePower = 1;
```

Comment:
```suggestion
				this.maxMove.basePower = 1;
```

(Also do this for all the other `maxMove!.basePower` assignments.)

I don't know if this was necessary before, but it won't be necessary after the above changes.